hunts/templates/all_puzzles_react.html

```diff
@@ -11,4 +11,50 @@
     var CURRENT_HUNT_ID = ({{ hunt_pk }}).toString();
 </script>
 <script src="{% static "main.js" %}"></script>
+
+
+<script>
+    function connect() {
+
+        var loc = window.location
+        var wsStart = "ws://"
+        if (loc.protocol == "https:"){
+            wsStart = "wss://"
+        }
+        var webSocketEndpoint =  wsStart + loc.host + '/notifications/' + ({{ hunt_pk }}).toString()
+
+        var socket = new WebSocket(webSocketEndpoint)
+
+        var audio = null;
+
+        socket.onmessage = function(e){
+            if (audio != null){
+                audio.pause()
+            }
+
+            audio = new Audio(e.data.replace(/^"(.*)"$/, '$1'));
```

Comment:
e.data is a quoted string, like `"\"data\""` so this removes `\"`'s. I don't know if there's a better way to do this to avoid the extra quotes, was hoping someone better at javascript could tell me if there is something better.

Reply:
Oh, I missed the quotes earlier (I had parsed them as string demarcators, not literal quotes).

I guess this is fine. Seems cleaner than doing substring(1, len-1)

Reply:
What's an example object? You might want `JSON.parse(e.data)` potentially